.travis.yml

```
@@ -1,6 +1,6 @@
 language: go
 go:
-- '1.11'
+- '1.11.4'
```

Not sure if multiline suggestions work...

```suggestion
- '1.11.4'
cache:
  directories:
  - $GOPATH/pkg/mod
```